models/issue_reaction.go

```diff
@@ -58,20 +59,28 @@ func FindCommentReactions(comment *Comment) (ReactionList, error) {
 }
 
 // FindIssueReactions returns a ReactionList of all reactions from an issue
-func FindIssueReactions(issue *Issue) (ReactionList, error) {
+func FindIssueReactions(issue *Issue, listOptions ListOptions) (ReactionList, error) {
 	return findReactions(x, FindReactionsOptions{
-		IssueID:   issue.ID,
-		CommentID: -1,
+		ListOptions: listOptions,
+		IssueID:     issue.ID,
+		CommentID:   -1,
 	})
 }
 
 func findReactions(e Engine, opts FindReactionsOptions) ([]*Reaction, error) {
```

Comment:
@SpaWn2KiLl if you name ther return variables then you dont need https://github.com/go-gitea/gitea/pull/9452/files#diff-61ce589b0f29ee34fc2448fcda3af467R82 and https://github.com/go-gitea/gitea/pull/9452/files#diff-61ce589b0f29ee34fc2448fcda3af467R78

and i also would remove redundancy https://github.com/go-gitea/gitea/pull/9452/files#diff-61ce589b0f29ee34fc2448fcda3af467R79

Reply:
dont think this has significant performance improvements if you aloc mem calculated based on pagination